backend/municipal-services/ingestion-service/app/utils/facility_service_client.py

```
@@ -1,13 +1,14 @@
 from typing import Dict, Any
 
 import requests
+from sqlalchemy import null, true
```

Ruff (0.8.2)

`sqlalchemy.null` imported but unused

Remove unused import

(F401)